packages/app/src/store/Context.tsx

```diff
@@ -193,6 +196,7 @@ export const PractitionerProvider: FC = () => {
         resourceType: resourceType,
       });
       const provider: Provider = {
+        email: profile?.telecom && profile.telecom[0].value ? profile.telecom[0].value : '',
```

Comment:
Seems like these property extractions are getting duplicated at least twice across the codebase. Maybe we should write a couple helper functions that we can then test/rely on in the future for parsing out specific FHIR properties?

Reply:
I encountered a bug where my provider's title was missing. Could be related to these types of extractions having some flaw in the logic.